SCAutolib/controller.py

```diff
@@ -533,7 +538,7 @@ def _general_steps_for_ipa():
         :return: name of the IPA client package for current Linux
         """
         os_version = _get_os_version()
-        if os_version not in (OSVersion.RHEL_9, OSVersion.CentOS_9):
+        if os_version in (OSVersion.RHEL_8, OSVersion.CentOS_8, OSVersion.Fedora):
             run("dnf module enable -y idm:DL1")
             run("dnf install @idm:DL1 -y")
             logger.debug("idm:DL1 module is installed")
```

Comment:
I do not think this this will work in Fedora. Fedora does not have IDM modules (but the failure might have been ignored in past -- I would suggest to skip this for Fedora)

Reply:
You are right, I removed it but I am not sure what it breaks and what it should be replaced with, more testing is needed.

Reply:
This should be just skipped on Fedora. In Fedora, all the packages are in the default repositories. This is needed only on RHEL8, where some stuff is coming from specific module.